packages/subgraph/tests/utils/utils.test.ts

```diff
@@ -0,0 +1,36 @@
+import {BigInt} from '@graphprotocol/graph-ts';
+
+import {assert, describe, test} from 'matchstick-as/assembly/index';
+import {bigIntToBytes32} from '../../src/utils/bytes';
+import {
+  ZERO,
+  ONE,
+  TWO,
+  MAX_UINT256_NUMBER_STRING,
+  ZERO_BYTES32,
+  ONE_BYTES32,
+  HALF_UINT256_BYTES32,
+  MAX_UINT256_BYTES32
+} from '../constants';
+import {TestCase} from './utils';
+
+describe('Test bytes', function() {
+  test('`bigIntToBytes32` with a range of `bigInt`s', function() {
```

Comment:
Personally, I prefer having one test case per test. Makes it sometimes easier to debug and finding some weird edge case bugs

Reply:
amended